actions/handle_caches/action.yaml

```diff
@@ -7,6 +7,9 @@ outputs:
   cache-libint:
     description: "Cache libint status"
     value: ${{ steps.cacheLibint.outputs.cache-hit }}
+  cache-libfort:
+    description: "Cache libfort status"
+    value: ${{ steps.cacheLibfort.outputs.cache-hit }}
```

Comment:
Again this seems like there's got to be a more general solution than per-dependency.

Reply:
I am not sure whether a more general solution can be found. I can try. But what I understood so far is that github maintains a boolean type variable "cache-hit" to indicate whether a cached version of a certain dependency can be found. So, I am not sure how we can have a single boolean variable for all dependencies. Let me see if they have a boolean array or something like that as the "cache-hit" variable. But I am not sure.